src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -1,5 +1,6 @@
 package mate.academy.spring.controller;
 
+import javax.persistence.EntityNotFoundException;
```

Comment:
Consider using DataNotFoundException or NoSuchElementException instead of EntityNotFoundException as it is more appropriate for data retrieval issues in the service layer.